.circleci/config.yml

```diff
@@ -466,12 +469,14 @@ jobs:
           name: Install dependencies
           command: yarn --immutable
       - save_cache:
-          key: dependency-cache-{{ checksum "/tmp/YARN_VERSION" }}-{{ checksum "yarn.lock" }}
+          key: dependency-cache-{{ checksum "/tmp/YARN_VERSION" }}-{{ checksum "yarn.lock" }}-{{ checksum "/tmp/FOUNDRY_VERSION" }}
           paths:
             - .yarn/cache
+            - .metamask/cache # should match yarn's relative location
       - persist_to_workspace:
           root: .
           paths:
+            - .metamask/cache # ensures anvil is installed
```

Comment:
Now that I'm doing a self review, I'm thinking this probably isn't required, since the `anvil` binary we need is symlinked to `.metamask/cache` from within in `node_modules/.bin`, and I think `persist_to_workspace` will copy the symlinks automatically when it copies `node_modules`.

Can't hurt to keep it though?